### ISSUE TITLE
Uncaught Error: ENOENT: no such file or directory

### ISSUE THREAD

**Issue by aseeeem** [mishak87/open-no-binaries#1]
Copy and pasted this from the editor. Happens immediately after launch.

[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 1.5.3
**System**: Linux Mint
**Thrown From**: [open-no-binaries](https://github.com/mishak87/open-no-binaries) package, v0.1.0
### Stack Trace

Uncaught Error: ENOENT: no such file or directory, open '/home/asim/Documents/pyhtonFiles'

```
At fs.js:549

Error: ENOENT: no such file or directory, open '/home/asim/Documents/pyhtonFiles'
    at Error (native)
    at Object.fs.openSync (fs.js:549:18)
    at Object.module.(anonymous function) [as openSync] (ATOM_SHELL_ASAR.js:137:20)
    at Function.module.exports.sync (/home/asim/.atom/packages/open-no-binaries/node_modules/isbinaryfile/index.js:116:25)
    at /home/asim/.atom/packages/open-no-binaries/lib/open-no-binaries.coffee:12:23
    at Workspace.module.exports.Workspace.openURIInPane (/opt/atom/resources/app.asar/src/workspace.js:465:22)
    at Workspace.module.exports.Workspace.open (/opt/atom/resources/app.asar/src/workspace.js:404:19)
    at AtomEnvironment.module.exports.AtomEnvironment.openLocations (/opt/atom/resources/app.asar/src/atom-environment.js:1038:19)
    at EventEmitter.outerCallback (/opt/atom/resources/app.asar/src/application-delegate.js:244:18)
    at emitTwo (events.js:92:20)
    at EventEmitter.emit (events.js:172:7)
```
### Commands

```
```
### Config

``` json
{
  "core": {
    "themes": [
      "atom-material-ui",
      "atom-material-syntax"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
atom-material-syntax, v0.4.3
atom-material-ui, v1.2.0
autoclose-html, v0.22.0
open-no-binaries, v0.1.0
remote-edit, v1.8.24

# Dev
No dev packages
```

**Comment by mishak87** [comment]
Thanks for report i will fix it tomorrow!

**Comment by aseeeem** [comment]
No problem! Good luck!

**Comment by mishak87** [comment]
Sorry for the delay 0.1.1 release includes potential fix.
Thanks for reporting!

Please confirm it fixes your issue. Thanks!

**Comment by mishak87** [comment]
Should be fixed in 0.1.1 if not please reopen. Thanks!
